### ISSUE TITLE
nvda does not read cell edits in google spreadsheets using microsoft hedge

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#8501]
### Steps to reproduce:
open a spreadsheet in google spreadsheets using microsoft hedge.
type a value in a cell.
Now focus the cell with the content, press f2 to edit the cell, and try to read the characters with right and left arrows.
### Actual behavior:
nvda does not even read it.
### Expected behavior:
nvda should read the characters you typed in the cell during editing.
in other browsers like firefox and chrome the cell edit reading is possible.
#### NVDA Installed/portable/running from source:
instaled
#### NVDA version:
alfa 15495
#### Windows version:
10 17134.165
#### Name and version of other software in use when reproducing the issue:
microsoft hedge
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no

**Comment by seanbudd** [comment]
is this bug still an issue in modern edge (based on chromium)?

**Comment by Adriani90** [comment]
@fernando-jose-silva a reminder to respond to the comment above.
Could you test if this is still reproducible for you with NVDA 2023.1 or with NVDA last alpha?

**Comment by fernando-jose-silva** [comment]
no, thank you very much, problem solved.

**Comment by Adriani90** [comment]
Thanks for the update, closing as works for me.
